src/pages/es/core-concepts/astro-components.md

```diff
@@ -297,15 +297,15 @@ Sin embargo, si deseas que todos tus scripts se extraigan de los componentes y s
 Un **script izado** se ve así:
 
 ```astro
-<script hoist>
+<script>
   // Una secuencia de comandos en línea
```

Comment:
Same here: looks like the `hoist` was added at some point, but the comment wasn’t updated.